### ISSUE TITLE
libatomic.so.1 not found

### ISSUE THREAD

**Issue by dzlandis** [NixOS/nixpkgs#142065]
The reason for suggesting this package is because some nodejs packages require it for some reason.

**Project description**
Support library providing __atomic built-in functions. 
When an atomic call cannot be turned into lock-free instructions, GCC will make calls into this library.

**Metadata**

* homepage URL: https://pkgs.org/download/libatomic1
* source URL:  http://archive.ubuntu.com/ubuntu/pool/main/g/gcc-10/libatomic1_10-20200411-0ubuntu1_amd64.deb
* license: N/A
* platforms: linux

**Comment by dzlandis** [comment]
> Have you tried `gcc.cc.lib`?

Yep, unfortunately that doesn't work either. I get the same error.

**Comment by illdefined** [comment]
> > Have you tried `gcc.cc.lib`?
> 
> Yep, unfortunately that doesn't work either. I get the same error.

Could you provide a bit more context? What kind of node.js package are you trying to build?

**Comment by dzlandis** [comment]
I was actually experimenting with [vosk](https://github.com/alphacep/vosk-api) which is a transcription package/api. Supposedly, it needs this libatomic.so.1 package to load voice models.

**Comment by Artturin** [comment]
where are you adding `gcc.cc.lib`

**Comment by dzlandis** [comment]
> where are you adding `gcc.cc.lib`

I'm actually just doing things through Replit's Nix system.
There's a replit.nix file which allows adding packages and so it looks like this:
```{ pkgs }: {
deps = [
    pkgs.nodejs-16_x
    pkgs.python39
    pkgs.gcc.cc.lib
	];
}
```
By all means though, I could be doing this completely wrong since I have very little knowledge about Nix in general so please let me know if that is the case.
